src/App.tsx

```diff
@@ -1,10 +1,22 @@
-import React from 'react';
+import { useState, useEffect } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [key, setKey] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const pressedKey = (event: KeyboardEvent) => {
+    setKey(() => event.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keyup', pressedKey);
+  });
```

Comment:
it's reasonable to cleanup before component unmounts
```suggestion
  useEffect(() => {
    document.addEventListener('keyup', pressedKey);
    
    return () => document.removeEventListener('keyup', pressedKey);
  }, []);
```